analysis/figures.Figure5.Rmd

```diff
@@ -416,20 +420,25 @@ sub_human_cite_pb_de <- FindMarkers(object = sub_human_cite_pb,
                          ident.1 = "Donor",
                          ident.2 = "AMI",
                          test.use = "DESeq2",
+                         assay = "RNA",
+                         slot = "counts",
                          min.pct = 0.1)
 
 sub_human_cite_pb_de$gene <- rownames(sub_human_cite_pb_de)
 sub_human_cite_pb_vwf <- subset(sub_human_cite_pb_de,gene == "VWF")
+
+## We will fetch the p-value for vWF as we are specifically looking at this one gene
 pvalue <- sub_human_cite_pb_vwf$p_val
 pvalue
 ```
 
+Finally, let's plot a violin plot for all cells across all samples in the endocardial cluster. First, we will plot a version with only two violins, one for each group.
 
 ```{r}
 Idents(sub_human_cite) <- sub_human_cite$disease_group
 vwf_vlnpot <- SCpubr::do_ViolinPlot(sample = sub_human_cite, 
                       features = "VWF",
-                      group.by = "disease_group",
+                      group.by = "disease_gro up",
```

Comment:
Is the space in "disease_gro up" on purpose?

Reply:
No that was obviously a type 😅 !